skimage/feature/texture.py

```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
     R : float
         Radius of circle (spatial resolution of the operator).
     method : {'default', 'ror', 'uniform', 'var'}
```

Comment:
```suggestion
    method : {'default', 'ror', 'uniform', 'nri_uniform', 'var'}
```
According to what follows...